src/components/AssetCheckoutScanner.tsx

```
@@ -0,0 +1,80 @@
+import { useEffect, useState } from 'react';
```

ESLint

[error] Replace `'react'` with `"react"`

(prettier/prettier)